iomgr/lib/dbengine/parser/AntlrHelpers.h

```diff
@@ -24,6 +24,14 @@ inline std::string& unquoteString(std::string& s)
 {
     s.pop_back();
     s.erase(0, 1);
+
+    size_t position = 0;
+    while (position != std::string::npos) {
+        position = s.find("''", position);
+        if (position == std::string::npos) continue;
+        s.erase(position, 1);
+        position++;
```

Comment:
code style: use prefix increment everywhere, unless you really need previous value from postfix increment